components/inputs/test/input-label.vdiff.js

```diff
@@ -19,18 +15,10 @@ const fieldsetTag = defineCE(class extends LitElement {
 		return { required: { type: Boolean } };
 	}
 
-	static get styles() { return [radioStyles]; }
-
 	render() {
-		const name = getUniqueId();
 		return html`
 			<d2l-input-fieldset label="Bun" ?required="${this.required}">
-				<label class="d2l-input-radio-label">
-					<input type="radio" name="${name}" checked>Whole wheat
-				</label>
-				<label class="d2l-input-radio-label">
-					<input type="radio" name="${name}">White
-				</label>
```

Comment:
Ended up removing checkboxes and radios from these "label" visual diffs. They have their own tests and it's not amazing to mix them into here.